File: test/fixtures/php-guzzle/commented_json_with_session.php
<?php
require 'vendor/autoload.php';

use GuzzleHttp\Client;

$client = new Client();

$response = $client->post('http://localhost:28139', [
    'headers' => [
        'Content-Type' => 'application/json',
        'Accept'       => 'application/json'
    ],
    // 'body' => '{   }',
    'json' => new stdClass()
]);

File: test/fixtures/ocaml/commented_json_with_session.ml
open Lwt
open Cohttp
open Cohttp_lwt_unix

let uri = Uri.of_string "http://localhost:28139" in
let headers = Header.init ()
  |> fun h -> Header.add h "Content-Type" "application/json"
  |> fun h -> Header.add h "Accept" "application/json" in
let body = Cohttp_lwt.Body.of_string "{   }" in
Client.post ~headers ~body uri
>>= fun (resp, body) ->
  (* Do stuff with the result *)
